### PR TITLE
CPM Python Package Release failure fix

### PR DESCRIPTION
**Pull request by Shamkh**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/communication/azure-communication-messages/tests/test_messages_client_async.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ async def test_image_send_message_async(self):
     @recorded_by_proxy_async 
     async def test_download_media_async(self):
         phone_number: str = "+14254360097"
-        input_media_id: str = "315a0b04-d80d-4573-bcf1-1ce33d3f19f0"
+        input_media_id: str = "8f8c29b2-c2e4-4340-bb28-3009c8a57283"
```
Comment (xiangyan99) [inline sdk/communication/azure-communication-messages/tests/test_messages_client_async.py:159]: Is this secret that we need to sanitize?

Comment (Shamkh) [reply sdk/communication/azure-communication-messages/tests/test_messages_client_async.py:159]: Hi @xiangyan99 , This is GUID that uniquely identifies media message image and expiry in 30 days. So this is not a secret.

Comment (xiangyan99) [reply sdk/communication/azure-communication-messages/tests/test_messages_client_async.py:159]: (I believe you already did that :)) please double check that they are already invalid.

**Review by xiangyan99 — APPROVED** [review]

File: sdk/communication/azure-communication-messages/tests/test_messages_client_async.py (hunk 1 above)
Comment (Shamkh) [reply sdk/communication/azure-communication-messages/tests/test_messages_client_async.py:159]: "RequestUri": "https://sanitized.unitedstates.communication.azure.com/messages/streams/sanitized?api-version=2024-02-01",
Added it.

**Pull request merged**